react/ChallengeTradePolicyCondition.tsx

```diff
@@ -85,8 +86,23 @@ const isProfileAllowed = (sessionResponse: SessionResponse | undefined) => {
   return 'unauthorized'
 }
 
+const isTradePolicyAllowed = (sessionResponse: SessionResponse | undefined) => {
+  if (sessionResponse == null) {
+    return null
+  }
+
+  const hasAccessToTradePolicy = (sessionResponse as Session).namespaces?.store?.channel.value
+
+  if(hasAccessToTradePolicy === "1"){
+    return true
+  }
+
+  return false
```

Comment:
da pra simplificar aqui
```suggestion
  return hasAccessToTradePolicy === "1"
```